mls-rs/test_harness_integration/src/main.rs

```diff
@@ -734,6 +743,7 @@ impl MlsClientImpl {
 
         let roster = group.roster().members();
 
+        let group_clone = group.clone();
```

Comment:
This clone seems useless?

Reply:
Unfortunately it is required because once you create a builder or processor that references `&mut Group` then you can no longer call any functions on Group until you complete the building process. It's actually a problem I hit in a few places within tests, and I think we will have to separate out the structures in terms of parsing vs the actual building so that the &mut reference is not an issue.